Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+#if os(iOS) && !SENTRY_NO_UIKIT
+@_implementationOnly import _SentryPrivate
+import UIKit
+
+/**
+ * A user feedback item that serializes to an envelope with the new format described at
+ * https://develop.sentry.dev/application/feedback-architecture/#feedback-events.
+ */
+@objcMembers
+class SentryFeedback: NSObject, SentrySerializable {
+    enum Source: String {
+        case widget
+        case custom
+    }
+
+    var name: String?
+    var email: String?
+    var message: String
+    var hints: [String: Any]?
+    var source: Source
+    let eventId: SentryId
+    
+    /// The event id that this feedback is associated with, like a crash report.
+    var associatedEventId: String?
+
+    init(name: String?, email: String?, message: String, hints: [String: Any]? = nil, source: Source = .widget, associatedEventId: String? = nil) {
```

Comment:
`l`: I would put the required message first and set the others to = nil per default, so the API is a bit cleaner.
```suggestion
    init(message: String, name: String? = nil, email: String? = nil, hints: [String: Any]? = nil, source: Source = .widget, associatedEventId: String? = nil) {
```